examples/Makefile

```diff
@@ -0,0 +1,15 @@
+REPOROOT=../
+# Use make help, to see the available rules
+
+include ../.make.defaults
+
+#venv::	.transforms.venv 	.defaults.lib-src-venv
```

Comment:
You may want to add a `clean` target to remove the venv (I think you could use .defaults.clean)